webui/src/Emulator/Emulator.tsx

```diff
@@ -22,6 +22,8 @@ import {
 	EmulatorImageCache,
 } from '@companion-app/shared/Model/Common.js'
 import { Operation as JsonPatchOperation } from 'fast-json-patch'
+import { UserConfigStore } from '../Stores/UserConfigStore.js'
+import { useUserConfigSubscription } from '../Hooks/useUserConfigSubscription.js'
 
 export function Emulator() {
```

Comment:
```suggestion
export const Emulator = observer(function Emulator() {
```

This needs wrapping in observer, so that it re-renders whenever the watched properties change